.gitbook/FAQ.md

```diff
@@ -147,7 +147,7 @@ Boba network is a permission-less network and hence we cannot influence app crea
 
 Unfortunately no, not at the moment.
 
-## Could you please explain the difference between the two mappings [mentioned in the token list](https://docs.boba.network/developer-docs/exchange-integration#the-boba-token-list?)?
+## Could you please explain the difference between the two mappings [mentioned in the token list](https://docs.boba.network/developer/exchange-integration#the-boba-token-list?)?
```

Comment:
Similar to above, shouldn't this be relative? Also, the extra `?` keeps this anchor from resolving.